### PR TITLE
Partner support update 071816

### PR DESCRIPTION
**Pull request by BryGittyUp**
Multiple changes to update e-mails from T3N to ctl.io.  Also updated security section.

### CODE REVIEW

**Comment by srveit** [comment]
@BryGittyUp please fix merge conflicts. Contact me if you need help.

Also, what is your Slack handle?

**Comment by BryGittyUp** [comment]
I may need assistance or a bit of guidance. Sorry, I'm new to using GitHub and it's not making a ton of sense to me. On Slack I'm @bryanc.  Sorry, haven't used that much either.

File: Partners/support-escalation-process-for-partners.md [hunk 1]
```diff
@@ -138,12 +138,16 @@ The following table describes list of operational support activities and request
...
 </table>
 
-### Ticket Prioritization Matrix
+### Ticket Prioritization Matrix and Escalations
 
 Tickets are worked according to priority in the order in which they are received. The priority designation indicates the level, extent, and impact on the partner.
 
 [Learn More About Ticket Prioritization](../Support/ticket-prioritization-matrix.md)
 
+If after reviewing hte Ticket Prioritization Matrix you need to escalate a ticket, the following articel details the escalation process.
```
Comment (chrislittle) [inline Partners/support-escalation-process-for-partners.md]: check spelling 'hte ticket'

File: Partners/support-escalation-process-for-partners.md [hunk 2]
```diff
@@ -199,22 +196,12 @@ These steps sit inside the “Restoration Process” within the overall Incident
...
-      <td>Security Management Escalation (Warning – Do Not Abuse)</td>
-      <td>Security Management Engagement</td>
-      <td>sec-esc@t3n.zendesk.com + Escalation ticket #</td>
+      <td>escalation@ctl.io</td>
     </tr>
     <tr>
       <td>Executive Escalation (Warning – Do Not Abuse)</td>
       <td>Executive Engagement – Enterprise Support Level Only</td>
-      <td>exec-esc@t3n.zendesk.com + Management Escalation ticket #</td>
+      <td>execescalation@ctl.io + Management Escalation ticket #</td>
```
Comment (chrislittle) [inline Partners/support-escalation-process-for-partners.md]: not sure this should be public,  escalation is done via a formal process not customer emailing executives (which pages them) directly.  please align with support process.

File: Partners/support-escalation-process-for-partners.md [hunk 3]
```diff
@@ -199,22 +196,12 @@ These steps sit inside the “Restoration Process” within the overall Incident
     <tr>
```
Comment (rpdubose) [inline Partners/support-escalation-process-for-partners.md]: We need to use markdown tables.

File: Partners/support-escalation-process-for-partners.md (hunk 1 above)
Comment (BryGittyUp) [inline Partners/support-escalation-process-for-partners.md]: Corrected spelling and removed Exec Escalation e-mail from below.

**Comment by srveit** [comment]
Closed. Will make new PR.

**Pull request closed without merging**